src/enip/cip/connection-manager/index.js

```diff
@@ -0,0 +1,197 @@
+/**
```

Comment:
Thanks for these references! :rocket:

Reply:
The documentation of cip is pretty good imo.

Reply:
It is once you get used to it lol... I am attempting to get my company to buy me the updated spec which is supposed to much better than the documents I have been working from in the manuals folder of this project.